### ISSUE TITLE
Add info box with a message regarding performed auto-upgrade process for pro apps

### ISSUE THREAD

**Issue by petrenkonikita112263** [xwikisas/application-licensing#153]
Currently, the License application provides the availability to set Auto Upgrade, which will trigger a certain job for checking the newest version for pro apps.  Enabling it removes the need for the manual check and update process from the Extension Manager.

![guupjgYiFQ](https://github.com/xwikisas/application-licensing/assets/56109799/b254ebbf-8f69-4202-bdd7-5ff7ac8fd064)

The result is that pro apps will have a recent version.

However, all this job is performed in the background, and it's hard to trace the operation of what exactly pro apps were updated and when exactly, even for advanced users or admin users of XWiki. What can be done - to display the info box under or below the license table with the message "**The extension {name} was upgraded from {old_version_number} to {new_version_number} at {date}**" a bit similar to what Extension Manager provides on the execution of XWiki "**Indexed on {date}**".

**Comment by oanalavinia** [comment]
Hi!  I'm not sure if this is a valid improvement since what you are describing it's already implemented. Currently, there is a notification send when an upgrade is done, only that users need to opt in for it (it cannot be done by us) . Check https://github.com/xwikisas/application-licensing/issues/88#issuecomment-777341542 for more details
What I think it would be best, is to have some information about it somewhere else then on the issue, since it happened before as well that users didn't knew about this feature

**Comment by petrenkonikita112263** [comment]
Yes, I checked it and saw in the code where this line is represented. But by default, the notifications regarding auto upgrade are disabled and not many users (even admins) enabled it. I can't find any mention in documentation that it should be enabled after auto-upgrade is enabled or not.

**Comment by oanalavinia** [comment]
To be more precise, the auto-upgrades feature does not have documentation somewhere else than on the issue that added this feature #88 and this is the real problem.
Indeed, because of this lack of documentation, the `Auto Upgrade` button from the Licenses page can be misleading since you expect that enabling it will also imply a notification when something is done. And since users will not look for the issue where the feature was added, they will not know what to do. 
So, what I propose is to update this issue to be about making these configurations known to the user. WDYT @petrenkonikita112263 , it's ok if I change it?

**Comment by petrenkonikita112263** [comment]
Okay, I agree with mentioned suggestion and you can change the title of this issue.
